rust/read_file.rs

```diff
@@ -0,0 +1,21 @@
+use std::env;
+use std::fs::File;
+use std::io::{BufReader, BufRead};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
```suggestion
fn main() {
```

Instead of the `-> Result<(), Box<dyn std::error::Error>>` approach, we'll just use `.unwrap();` instead of `?;`